release-notes/v1_92.md

```diff
@@ -498,6 +498,14 @@ In this milestone, we picked up the work again to adopt ESM for VS Code core. Ou
 
 The new scroll bar in the terminal required a significant amount of behind-the-scenes work. The terminal in VS Code is built on the [xterm.js OSS project](https://github.com/xtermjs/xterm.js), and in its upcoming release will also ship a small portion of VS Code's codebase, specifically part of the `base/` folder.
 
+### Electron 30 update
+
+In this milestone, we are promoting the Electron 30 update to users on our stable release. This update comes with Chromium 124.0.6367.243 and Node.js 20.14.0. We want to thank everyone who self-hosted on Insiders builds and provided early feedback.
+
+**Notice of breaking API change when spawning `.bat` or `.cmd` files:**
+
+The Node version with this Electron update contains a [breaking change](https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2#command-injection-via-args-parameter-of-child_processspawn-without-shell-option-enabled-on-windows-cve-2024-27980---high), in response to a CVE, which may affect you if you execute `.bat` or `.cmd` files on Windows. You can follow the [guidelines](https://nodejs.org/api/child_process.html#spawning-bat-and-cmd-files-on-windows) set by Node.js when spawning these files using the `shell` option. We have tried to avoid any surprises for our extension community by proactively notifying of this change by a simple scan of the extension source code https://github.com/aws/aws-toolkit-vscode/issues/5292, https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/920, https://github.com/dotnet/vscode-dotnet-runtime/issues/1870. It is possible we might have not caught all the impacted extensions if the extension source does not deal with these files but might have an indirect dependency at runtime. We recommend extension authors to always test using our [insiders version](https://code.visualstudio.com/insiders/) to catch these changes in advance.
```

Comment:
I would replace

> We have tried ... dependency at runtime

with

> We have proactively notified extensions who may be affected based on a simple source code scan, but it's possible with may have missed yours. If you are affected, you will encounter an `EINVAL` exception when spawning `.bat` or `.cmd` files on Windows. Please refer to the [Node.js documentation](https://nodejs.org/api/child_process.html#spawning-bat-and-cmd-files-on-windows) for guidance on how to handle these.

I worry that linking to specific issues on other repos will lead to confused people looking for help or bothering other maintainers

Reply:
+1 on Connor's remarks.

 There's a small typo:
> it's possible with may have missed yours -> it's possible **we** may have missed yours

Reply:
👍🏻 Wow, that’s very thoughtful and considerate!